classes/invitation/invitations/userRoleAssignment/resources/UserRoleAssignmentInviteResource.php

```diff
@@ -65,4 +67,40 @@ public function toArray(Request $request)
             'newUser' => $this->transformUser($newUser),
         ]);
     }
+
+    public function transformInvitationPayload($userId, array $payload, Context $context)
```

Comment:
Revisiting this, can't we use the $invitationPayload object instead of an array? If no, what prevents us from doing that?